presto-docs/src/main/sphinx/overview/concepts.rst

```diff
@@ -183,6 +183,32 @@ that statement. A query encompasses stages, tasks, splits, connectors,
 and other components and data sources working in concert to produce a
 result.
 
+Query Plan
+^^^^^^^^^^
+
+A query plan is a sequence of steps used to access and manipulate data 
+according to the SQL query. It is represented as a tree of nodes, with 
```

Comment:
@steveburnett : A query plan is loosely an operator. The mapping is not necessarily 1:1. PlanNodes are just logical or physical plannodes that represent a logical unit of relational algebra that can be executed by the. engine.

Reply:
Thanks @aditi-pandit! I revised the description of query plan using your comment. Please review when you can.

Reply:
The plan itself is not an operator, the plan nodes are. So the documentation provided by @tdcmeehan is more apt (with addition of the `loosely` ) -  

>A query plan is a sequence of steps used to access and manipulate data according to the SQL query. It is represented as a tree of nodes, with each node **loosely** representing an [operator](https://prestodb.io/docs/current/overview/concepts.html#operator).